src/jormungandr/testing/jormungandr-integration-tests/src/networking/p2p/public_traffic.rs

```diff
@@ -94,8 +94,8 @@ fn public_gossip_rejection() {
     assert!(gossip_dropped);
 }
 
-#[ignore]
 #[test]
+#[ignore]
 pub fn test_public_node_cannot_publish() {
```

Comment:
why this tests are in ignore?It is done on purpose?

Reply:
Yes, mainly because each test takes over 1 minute.